tests/libvmifact/factory.go

```diff
@@ -97,28 +95,11 @@ func NewAlpineWithTestTooling(opts ...libvmi.Option) *kvirtv1.VirtualMachineInst
 
 func NewGuestless(opts ...libvmi.Option) *kvirtv1.VirtualMachineInstance {
 	opts = append(
-		[]libvmi.Option{libvmi.WithResourceMemory(qemuMinimumMemory())},
+		[]libvmi.Option{libvmi.WithResourceMemory(GetQemuMinimalMemory())},
```

Comment:
there's another usage of this logic in https://github.com/kubevirt/kubevirt/blob/main/tests/storage/restore.go#L1207 . Consider fixing it in another commit.